1 table.sql

```diff
@@ -0,0 +1,61 @@
+use BikeShopDB
+go
+
+drop table if exists Bike
+go 
+create table dbo.bike(
+    BikeId int primary key identity,
+    CustomerName varchar(50),
+    CustomerAddress varchar(150),
+    PhoneNumber varchar(20),
+    BikeCompany varchar(100) not null constraint ck_Bike_Company_cannot_be_blank check(BikeCompany <> ''),
+    BikeSize varchar(6) not null constraint ck_Bike_Size_cannot_be_blank check(BikeSize <> ''),
+    BikeColor varchar(15) not null constraint ck_Bike_Color_cannot_be_blank check(BikeColor <> ''),
+    DatePurchased date not null constraint ck_Date_Purchased_must_be_after_1st_of_march_2022 check(DatePurchased > '2022-03-01'),
```

Comment:
Make sure that DatePurchased is not in the future. This condition can be combined with the constraint below.